gme/Gb_Oscs.cpp

```diff
@@ -15,8 +13,6 @@ details. You should have received a copy of the GNU Lesser General Public
 License along with this module; if not, write to the Free Software Foundation,
 Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA */
 
-#include "blargg_source.h"
```

Comment:
Is this really not needed here? (Haven't read carefully this part)

Reply:
Maybe it was leftover from a time when it was needed.